docs/user-documentation/index.md

```diff
@@ -94,11 +94,10 @@ There are multiple Companion apps available:
   - Android: [GadgetBridge](https://gadgetbridge.org/)
   - SailfishOS: [Amazfish](https://openrepos.net/content/piggz/amazfish/)
   - iOS: [InfiniLink](https://github.com/xan-m/InfiniLink)
-  - PinePhone (Linux phone): [Siglo](https://github.com/alexr4535/siglo)
+  - PinePhone (Linux phone): [Siglo](https://github.com/alexr4535/siglo) and [WatchMate](https://github.com/azymohliad/watchmate)
 - Linux Computer:
-  - [Amazfish](https://openrepos.net/content/piggz/amazfish/) and
-    [Siglo](https://github.com/alexr4535/siglo) will also work, but may require
-    some manual installation.
+  - [Amazfish](https://openrepos.net/content/piggz/amazfish/),
+    [Siglo](https://github.com/alexr4535/siglo) and [WatchMate](https://github.com/azymohliad/watchmate)
   - [ITD](https://gitea.arsenm.dev/Arsen6331/itd/) 
```

Comment:
I think it would be good to replace this with something similar to the README, because the line between desktop and mobile isn't very clear-cut when it comes to Linux.

Reply:
I copied the lines from the README